### DISCUSSION TITLE
Patchcable include before getting used

### DISCUSSION THREAD

**Discussion by chbally** [netbox-community/netbox#14255] [Help Wanted!]
I ordered about 50 Patchcables in different sizes and labled them with a unique number on both ends.
Now I wanted to put them all in Netbox and found out, that without a Divice and Termination on both side I'm not able to put the cables into Netbox.
I'm not shure, whether I didn't see the right way to do so, this is featurerequest or documented somewhere else.
Any help is appreciated.

**Comment by gdprdatasubect** [comment]
Why would you need to track unused cables?
I personally label my cables with random strings and use that as a label in netbox when i patch

**Reply by chbally** [reply]
Mybe to know which length I have in storck udn what needs to be orderd.

**Reply by gdprdatasubect** [reply]
That seems more hassle than it's worth to me…

Simply counting your stock vs. creating them in a management system first.
At my workplace all two-digit-cost-items and lower three digits aren't even inventorized because updating these information is too expensive in regards of labour cost to be justifiable.
